cmd/kube-egress-gateway-cnimanager/cmd/serve.go

```diff
@@ -92,7 +93,10 @@ func ServiceLauncher(cmd *cobra.Command, args []string) {
 		logger.Error(err, "failed to create cni config manager")
 		os.Exit(1)
 	}
+	wg := sync.WaitGroup{}
```

Comment:
I'd prefer https://pkg.go.dev/golang.org/x/sync/errgroup But this is optional.

Reply:
thanks, please take another look at the errgroup usage.

Reply:
thanks, please take another look at the errgroup usage.